<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Spoti2Tidal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_root">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_main">
      <item>
       <widget class="QFrame" name="sidebarFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="minimumWidth">
         <number>240</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_sidebar">
         <item>
          <widget class="QLabel" name="appTitleLabel">
           <property name="text">
            <string>Spoti2Tidal</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="spotifyStatusLabel">
           <property name="text">
            <string>Spotify: Signed out</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spotifySignInButton">
           <property name="text">
            <string>Sign in to Spotify</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tidalStatusLabel">
           <property name="text">
            <string>TIDAL: Signed out</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tidalSignInButton">
           <property name="text">
            <string>Sign in to TIDAL</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="sidebarSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="transferButton">
           <property name="text">
            <string>Sync / Transfer</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_content">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="playlistsTab">
           <attribute name="title">
            <string>Playlists</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_playlists">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_spotify">
              <item>
               <widget class="QLabel" name="spotifyPlaylistsLabel">
                <property name="text">
                 <string>Spotify Playlists</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="spotifyPlaylistsView"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_tidal">
              <item>
               <widget class="QLabel" name="tidalPlaylistsLabel">
                <property name="text">
                 <string>TIDAL Playlists</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="tidalPlaylistsView"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tracksTab">
           <attribute name="title">
            <string>Tracks</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_tracks">
            <item>
             <widget class="QLabel" name="savedTracksLabel">
              <property name="text">
               <string>Spotify Saved Tracks</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="savedTracksTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="logsTab">
           <attribute name="title">
            <string>Activity / Logs</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_logs">
            <item>
             <widget class="QPlainTextEdit" name="logView">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_footer">
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="text">
             <string>Ready</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 </ui>


